HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1137,7 +1141,11 @@ def import(schedules_paths)
         end
 
         if @schedules.keys.include? col_name
-          fail "Schedule column name '#{col_name}' is duplicated. [context: #{schedules_path}]"
+          if col2path[col_name] == schedules_path
+            fail "Schedule column name '#{col_name}' is duplicated. [context: #{schedules_path}]"
+          else
+            @runner.registerWarning("Schedule column name '#{col_name}' already exist in #{col2path[col_name]}. Overwriting with #{schedules_path}.")
+          end
```

Comment:
If a column name shows up twice in the same file, it's an error, but if it shows up in different files, the last file is used? Do I have that right? I don't understand why we would want to treat the two situations differently.

Reply:
Yeah, you got that right. But I am unsure myself why I needed to add this. I will come back to this!